lib/block_view_types.go

```diff
@@ -3614,6 +3614,12 @@ type GlobalParamsEntry struct {
 
 	// The new minimum fee the network will accept
 	MinimumNetworkFeeNanosPerKB uint64
+
+	// MaxNonceExpirationBlockBuffer is maximum value nodes will
+	// tolerate for the difference between the current block height
+	// and the expiration block height specified in the nonce for a
+	// transaction.
+	MaxNonceExpirationBlockBuffer uint64
```

Comment:
Let's make this param name a bit more self documenting. Feel free to ignore if you disagree.
```suggestion
	MaxNonceExpirationBlockHeightOffset uint64
```